install/operator/pkg/util/kube.go

```diff
@@ -173,3 +180,33 @@ func Exec(o ExecOptions) error {
 	}
 	return exec.Stream(o.StreamOptions)
 }
+
+/**
+ * Find the pods that have been created by a Job
+ */
+func PodsFromJob(ctx context.Context, c client.Client, job *batchv1.Job, namespace string) (*corev1.PodList, error) {
+	selector := job.Spec.Selector
+	controllerUid := selector.MatchLabels[CONTROLLER_UID_LABEL]
+
+	podList := &corev1.PodList{}
+	err := c.List(ctx, podList,
+		client.InNamespace(namespace),
+		client.MatchingLabels{
+			CONTROLLER_UID_LABEL: controllerUid,
+			JOB_NAME_LABEL:       job.Name,
+		})
+
+	if err != nil {
+		return nil, err
+	}
+
+	return podList, nil
+}
```

Comment:
The method is called only once from `backup,go`, no reason for having it here instead of in the package where it is called from, actually it is totally fine not to have a method for this since it is called once.